src/handlebars/support.handlebars

```diff
@@ -59,6 +59,360 @@
         and we will work with you and the appropriate development team to resolve it.
         Any fixes we identify will be delivered as part of the next AdoptOpenJDK release.
       </p>
+	  
+		<h3 id="openjdk8_platforms">OpenJDK 8</h3>
+		<p>Where available, OpenJDK 8 binaries are supported on the minimum operating system levels shown in the following tables:</p>
+
+		<table border="1">
+			<thead>
+				<tr>
+					<th>Key</th>
+					<th>Value</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>HotSpot variant</td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span></td>
```

Comment:
I think the fa fa-start-o and style color of orange could all be wrapped up in own specific class?

Reply:
Simplified a little.